packages/user-group-spa/src/components/Form.jsx

```diff
@@ -0,0 +1,228 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useParams, useHistory } from 'react-router-dom';
+import {
+  Form,
+  FormGroup,
+  TextInput,
+  SearchInput,
+  ActionGroup,
+  Button,
+  // Bullseye,
+  // Spinner,
+  // SimpleList,
+  // SimpleListItem,
+  // ChipGroup,
+  // Chip,
```

Comment:
Are these really needed?

Reply:
Yes, that part of the code and other lines in that file are commented as a workaround until we have the Group Search API.

I kept it in because once we have that integration, we just have to uncomment those lines.